patrol/patrol.py

```diff
@@ -6,28 +7,27 @@
 class Patrol:
 
     def __init__(self, sentry_api_token, timeout=None):
+        self.headers = {
+            'Authorization': 'Bearer {}'.format(sentry_api_token)
+        }
+        self.timeout = timeout
         self.api = get_api_instance(sentry_api_token, timeout)
 
-    def events(self, organization, project):
-        all_events = []
-        events = self.api.project_events.list(organization, project)
-        all_events.extend(events.body)
-
-        links = parse_header_links(events.headers.get('Link'))
-        while links[1]['results'] == 'true':
-            events = self.api.project_events.list(organization, project)
-            all_events.extend(events.body)
+    def _fetch_resources(self, endpoint, organization, project):
+        endpoint = getattr(self.api, endpoint)
+        method = getattr(endpoint, 'list')
 
-        return all_events
+        resources = method(organization, project)
```

Comment:
could you try to return resource and use `resource.client_response`? it's the requests Response embedded in resource client response. I believe this way the rest of the code will be clearer... (then you can use `client_response.links`)

Reply:
done